lib/rspec/core/did_you_mean.rb

```diff
@@ -0,0 +1,44 @@
+module RSpec
+  module Core
+    # Service object to provide did_you_mean suggestions
```

Comment:
Unless you make this object yard private `# @private` you'll need to document all the methods in this file with yard, but I'm happy for this to be a private API. I'd suggest an improvement to this comment however.
```
# Wrapper around Ruby's `DidYouMean::SpellChecker` when available to provide file name suggestions.
```
As all of RSpec is "service" objects., and `did_you_mean` isn't a code path for us :)

Reply:
done